__tests__/api.test.js

```diff
@@ -165,3 +165,35 @@ describe("PATCH /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("returns a status of 200 and an array", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toBeInstanceOf(Array);
+      });
+  });
+  test("returns an array of user objects with correct key-value pairs", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user.username).toEqual(expect.any(String));
+          expect(user.name).toEqual(expect.any(String));
+          expect(user.avatar_url).toEqual(expect.any(String));
+        });
+      });
+  });
+  test("Returns a status of 404 when given path which does not exist", () => {
```

Comment:
This test isn't specific to /api/users so I'd make it a stand alone describe block